setup.cfg

```diff
@@ -33,7 +33,7 @@ python_requires = >=3.7
 install_requires =
     torch>=1.12
     bitsandbytes==0.40.0.post4
-    accelerate>=0.16.0,<1.0.0
+    accelerate>=0.16.0,<0.21.0
```

Comment:
accelerate 0.21.0 released yesterday imports bnb automatically if it's installed :( Maybe this will change in future.